acme/setup.py

```diff
@@ -31,7 +31,8 @@
 else:
     install_requires.append('mock')
 
-if sys.version_info < (2, 7, 9):
+# Installing for all 2.7 versions for wheel compatibility: #2175
```

Comment:
Please leave explicit inline comment reference to 2.7.8 as the last one requiring those packages. this comment should also be moved and merge with the one before branching condition